src/reference/asciidoc/file.adoc

```diff
@@ -616,6 +616,9 @@ outbound gateway by using the XML namespace.
 
 Starting with version 4.3, you can specify the buffer size to use when writing files.
 
+Starting with version 5.1, you can provide a `newFileCallback` which will be triggered if you are using `FileExistsMode.APPEND` or `FileExistsMode.APPEND_NO_FLUSH` and a new file has to be created.
```

Comment:
We need also to mention what type it is.

Reply:
type as in `BiConsumer<File, Message<?>>` ?

Reply:
Correct, but it is not reflected in the Docs. Isn't it?